### PR TITLE
Container converter

### PR DESCRIPTION
**Pull request by tevoinea**
The container type wasn't converting properly to and from the DB. This fixes that.

### CODE REVIEW

**Review by nharper285 — APPROVED** [review]

**Review by stishkin — APPROVED** [review]

File: src/ApiService/ApiService/onefuzzlib/orm/EntityConverter.cs [hunk 1]
```diff
@@ -205,6 +205,11 @@ private EntityInfo GetEntityInfo<T>()
 
                 tableEntity.Add(prop.columnName, string.Join(",", values));
             }
+            else if (prop.type == typeof(Container))
```
Comment (chkeita) [inline src/ApiService/ApiService/onefuzzlib/orm/EntityConverter.cs:208]: don't couple the entity converter with the enties themselves.
instead, create jsonconverter on the entity type

Comment (chkeita) [reply src/ApiService/ApiService/onefuzzlib/orm/EntityConverter.cs:208]: `BaseEventConverter` is one example of jsonConverter. In your case you can create a converter and apply it to the `Container` type

File: src/ApiService/Tests/OrmTest.cs [hunk 2]
```diff
@@ -276,5 +276,25 @@ public void TestEventSerialization2()
...
         }
+
+        record Entity3(
+            [PartitionKey] int Id,
+            [RowKey] string TheName,
+            Container Container
+        ) : EntityBase();
+
+        [Fact]
+        public void TestContainerSerialization()
```
Comment (chkeita) [inline src/ApiService/Tests/OrmTest.cs:287]: This Test is probably going to work even without your change. 
The problem is the  default serialization will translate a container to an object 
``` json 
{
   "ContainerName": "<name>"
}
```

but in existing object a container is represented by a simple string 

This test make sure that we can deserialize what **WE** serialize but it does not show that w can deserialize existing objects

**Review by chkeita — CHANGES_REQUESTED** [review]
See comments

File: src/ApiService/ApiService/onefuzzlib/orm/EntityConverter.cs (hunk 1 above)
Comment (chkeita) [reply src/ApiService/ApiService/onefuzzlib/orm/EntityConverter.cs:208]: also this fix onlyworks when the entity is saved to the db but not for when we read a json string

**Review by chkeita — APPROVED** [review]

**Pull request merged**